internal/app/machined/pkg/runtime/v1alpha1/bootloader/mount/mount.go

```diff
@@ -56,13 +58,25 @@ func PartitionOp(
 					return xerrors.NewTaggedf[NotFoundTag]("partition %d with label %s is not of type %s (actual %q)", partition.PartitionIndex, *partition.PartitionLabel, spec.FilesystemType, partition.Name)
 				}
 
-				points = append(points,
-					mount.NewPoint(
-						partitioning.DevName(disk, partition.PartitionIndex),
-						spec.MountTarget,
-						spec.FilesystemType,
-						newPointOptions...,
-					),
+				manager := mount.NewManager(slices.Concat(
+					[]mount.ManagerOption{
+						mount.WithTarget(spec.MountTarget),
+						// mount.WithRepairOnUnmount(),
```

Comment:
is this commented code meant to be for something in the future?

Reply:
No, I missed that during cleanup. That was part of the debugging of the imager issue, both function and call will be dropped.

Thanks!

Reply:
Removed.